src/components/create-contingency-list-form.js

```diff
@@ -75,7 +81,7 @@ export const CreateContingencyListForm = (props) => {
 
         let isPrivateContingencyList = contingencyListPrivacy === 'private';
 
-        props.setOpen(false);
+        onClose();
```

Comment:
For homogenous code, you should move this call in a `if(res.ok)` in the Promise.then bellow.
Same thing that you did in create-study-form